server/model/monitor.js

```diff
@@ -809,8 +809,39 @@ class Monitor extends BeanModel {
                 bean.important = true;
 
                 if (Monitor.isImportantForNotification(isFirstBeat, previousBeat?.status, bean.status)) {
+                    // Change the notification msg if status becomes UP from DOWN
+                    // Adding msg for the downtime
+                    let downTimeInSeconds;
```

Comment:
```suggestion
                    let downTimeInSeconds = null;
```

Reply:
Why bother setting it to `null`? It is already `undefined` so it doesn't really make much of a difference.